common/config/rush/command-line.json

```diff
@@ -0,0 +1,18 @@
+{
+    "$schema": "https://developer.microsoft.com/json-schemas/rush/v5/command-line.schema.json",
+  
+    "commands": [
+        {
+            "commandKind": "bulk",
+            "name": "build",
+            "allowWarningsInSuccessfulBuild": true
```

Comment:
Required change driven by rush now failing if there are any warnings written to the console during the build